### DISCUSSION TITLE
How to judge whether C is an equality constraint or inequality one?

### DISCUSSION THREAD

**Discussion by styw1234** [yalmip/YALMIP#1450] [General discussion]
How to judge whether C is an equality constraint or inequality one?

For example:
a=sdpvar(1,5);
C=[];
C=[C, a>=0];
C=[C, a==2];
we can use the following command if we want to see the expression of C
sdisplay(sdpvar(C(1:end,:))

then how to judge whether the expression, e.g., C(1,:), is an equality constraint or inequality one?

**Comment by johanlofberg** [comment]
```
>> C
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
|   ID|                    Constraint|   Coefficient range|
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
|   #1|   Element-wise inequality 5x1|              1 to 1|
|   #2|       Equality constraint 1x5|              1 to 2|
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
>> is(C(1),'equality')

ans =

     0

```

**Reply by styw1234** [reply]
thank you!

I found that more information can be received with the following command:

lmiinfo(C(1))
